projects/igniteui-angular/src/lib/core/i18n/query-builder-resources.ts

```diff
@@ -35,26 +38,46 @@ export interface IQueryBuilderResourceStrings {
     igx_query_builder_filter_false?: string;
     igx_query_builder_filter_all?: string;    
     igx_query_builder_title?: string;
+    igx_query_builder_from_label?: string;
+    igx_query_builder_select_label?: string;
+    igx_query_builder_where_label?: string;
     igx_query_builder_and_group?: string;
     igx_query_builder_or_group?: string;
     igx_query_builder_end_group?: string;
     igx_query_builder_and_label?: string;
     igx_query_builder_or_label?: string;
+    igx_query_builder_switch_group?: string;
     igx_query_builder_add_condition?: string;
-    igx_query_builder_create_and_group?: string;
-    igx_query_builder_create_or_group?: string;
```

Comment:
Removing those is technically a breaking change, however small. The resource interfaces are public API, just sayin :)

Reply:
added migration in this PR - https://github.com/IgniteUI/igniteui-angular/pull/15381